user-manual/administer/settings.rst

```diff
@@ -1871,3 +1872,95 @@ setting.
+
+Edit the Privacy Policy
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Administrators should also review the wording of the new default Privacy Policy static page. This can be found in |gears| "Admin > Static Pages >Privacy Policy", or it can be accessed via the Quick links menu in the AtoM header. Click on "edit" at the bottom of the view page, make your desired edits, and select "save" to save your changes or "cancel" to return to the default text.
+
+Upgrades to 2.4.1
+----------------------
+
+After first upgrading the cookie notification banner will appear as follows:
+
+.. image:: images/gdpr-banner-pre-task.*
+   :align: center
+   :width: 80%
+   :alt: GDPR banner first access
+
+* Additionally, if no further action is taken, the new default Privacy policy
+  static page will not exist
+* If you don't intend to use the banner or need the Privacy policy static page,
+  you can disable the notification banner via |gears| *Admin > Settings > Privacy policy*
+* To create the Privacy policy static page and update the banner message, a
+  system administrator should run the following task from the root AtoM installation directory: ``php symfony tools:run lib/task/tools/addGdprSettings.php``
+
+Custom theme updates
+--------------------
+
+* If you have a custom theme and the file <code>scaffolding.less</code> has
+  been customized, this file will need to be updated. Changes are identified `here <https://goo.gl/d6HVVf>` - see the changes to <code>scaffolding.less</code>
+* If a theme has been customized, but the file "scaffolding.less" is being
+  referenced from the arDominion theme, then there are no modifications required.
+  - This can be checked in your theme's
+    <code>plugins/<themefolder>/css/main.less</code> file.
+    Look for: <code>@import "../../arDominionPlugin/css/less/scaffolding.less";</code> - if this line is present, no modifications should be required
+* Check if <code>_header.php</code> has been overridden in the custom theme. If
+  so, the change highlighted in `this diff <https://github.com/artefactual/atom/commit/c65e84e809a5760c9814f8117a291bdb9a7491da#diff-e3a653026878cbc4745a5526934888d7will>` need to be made:
+* Post upgrade, run 'make' from your root theme plugin folder to rebuild the
+  CSS, and clear the application cache:
+  - If you are using the Dominion theme, run: ``make -C plugins/arDominionPlugin``
+  - If you are using a custom theme, then you might want to run the task above for Dominion, as well as for your custom theme: ``make -C plugins/<yourPluginDirectory``
+* And finally, in all cases, clear the application cache afterwards: ``php symfony cc``
+
+Upgrades to 2.5
+---------------
+
+No additional changes are required when upgrading to 2.5.x in the future. A migration will run, but it will find the GDPR settings already added during the upgrade to 2.4.1 and skip making any changes. If the settings did not happen to exist for any reason, they will be added by the 2.5.x migration script.
```

Comment:
Overall this is good, but admittedly different from what I had imagined. I had thought we'd simply add a warning admonition at the top mentioning the need for 2.4.1 users to see the release notes before using the feature, and proceed with just documenting the functionality. Instead, you've copied the whole upgrade instructions here... which works, but I think we should also include basic instructions on HOW to use the feature (i.e. as if it was all set up, no need to run the script, etc). Things like: 

* The setting can be turned on or off. By default upon installation, the banner will be set to off (this isn't the case right now, but we just re-opened the ticket and will be fixing this prior to the 2.4.1 release)
* The privacy page does not disappear when users turn the setting off. Users who don't want it should delete it. 
* It might end up being overkill, but simple step instructions for use? 

Just some thoughts! 

In any case, I would say don't bother cherry-picking this to 2.5, since so much of it is specific to 2.4 right now. We can just copy relevant sections and document it for 2.5 later.